requirements.txt

```diff
@@ -1 +1,3 @@
 cr8
+Cython
+asyncpg
```

Comment:
do we need to add the git link here so that the connection pool change is included?

Reply:
I don't think so, this seems to work with a clean venv